main.go

```diff
@@ -158,13 +162,22 @@ func main() {
 	globalHTTPTimeout := time.Duration(globalHTTPTimeoutMS) * time.Millisecond
 	eventRecorder := mgr.GetEventRecorderFor("keda-operator")
 
+	kubeClientset, _ := kubernetes.NewForConfig(ctrl.GetConfigOrDie())
```

Comment:
could you please use here the `cfg := ctrl.GetConfigOrDie()` defined on line 121?